setup_demo.yml

```diff
@@ -9,8 +9,16 @@
       loop: "{{ controller_components }}"
       vars:
         controller_components:
+          #- organizations
```

Comment:
Are these comments needed? Looks like it was corrected below

Reply:
They are. I should probably just remove and add during another PR